hoot-core-test/src/test/cpp/hoot/core/algorithms/SoundexTest.cpp

```diff
@@ -57,20 +53,20 @@ class SoundexTest : public CppUnit::TestFixture
 
   void runTest()
   {
-    CPPUNIT_ASSERT_EQUAL(QString("R163"), Soundex::encode("Robert"));
-    CPPUNIT_ASSERT_EQUAL(QString("R163"), Soundex::encode("Rupert"));
-    CPPUNIT_ASSERT_EQUAL(QString("R150"), Soundex::encode("Rubin"));
-    CPPUNIT_ASSERT_EQUAL(QString("A261"), Soundex::encode("Ashcraft"));
-    CPPUNIT_ASSERT_EQUAL(QString("A261"), Soundex::encode("Ashcroft"));
-    CPPUNIT_ASSERT_EQUAL(QString("T522"), Soundex::encode("Tymczak"));
-    CPPUNIT_ASSERT_EQUAL(QString("P236"), Soundex::encode("Pfister"));
+    HOOT_STR_EQUALS(QString("R163"), Soundex::encode("Robert"));
+    HOOT_STR_EQUALS(QString("R163"), Soundex::encode("Rupert"));
+    HOOT_STR_EQUALS(QString("R150"), Soundex::encode("Rubin"));
+    HOOT_STR_EQUALS(QString("A261"), Soundex::encode("Ashcraft"));
+    HOOT_STR_EQUALS(QString("A261"), Soundex::encode("Ashcroft"));
+    HOOT_STR_EQUALS(QString("T522"), Soundex::encode("Tymczak"));
+    HOOT_STR_EQUALS(QString("P236"), Soundex::encode("Pfister"));
```

Comment:
When `SoundexTest` wasn't part of the `hoot` namespace, `CPPUNIT_ASSERT_EQUAL` could compare two `QString`s but now it can't so we use the `HOOT_STR_EQUALS` macro like is was designed for.